crates/ty_python_semantic/src/types/ide_support.rs

```diff
@@ -155,6 +163,14 @@ impl<'db> AllMembers<'db> {
                 _ => {}
             },
 
+            Type::TypedDict(_) => {
+                if let Type::ClassLiteral(class_literal) = ty.to_meta_type(db) {
+                    self.extend_with_class_members(db, ty, class_literal);
+                }
+
+                self.extend_with_type(db, KnownClass::TypedDictFallback.to_instance(db));
+            }
```

Comment:
(You'll need to change this too if you think my comments above about the meta-type of `TypedDict`s is reasonable)